src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -192,8 +187,7 @@ public void AddCamera(Camera camera)
         /// Camera.Dispose() don't automatically release memory. We should call this API if we want to release memory.
```

Comment:
line 183:   "is set to" -> "becomes the"  or "in the list is selected."
about the remarks section: does this mean Camera.Dispose() is useless? That sounds bad.
 and the remarks section should be changed to use passive or "is ... / will be ..." statements instead of "we": "This method should be called....  (Default camera cannot be removed.) Attempt/request to remove default camera will be ignored."

Reply:
Thank you I fixed the comment.